src/functions/artifacts.spec.ts

```diff
@@ -0,0 +1,318 @@
+import { expect } from "chai";
+import * as sinon from "sinon";
+
+import * as artifactregistry from "../gcp/artifactregistry";
+import * as artifacts from "./artifacts";
+
+describe("functions artifacts", () => {
+  describe("makeRepoPath", () => {
+    it("should construct a valid repo path", () => {
+      const path = artifacts.makeRepoPath("my-project", "us-central1");
```

Comment:
IDK if I feel comfortable with this hard-coding the repository ID, especially since it might change with CRF

Reply:
I feel okay since this is documented in GRF docs and has not changed since beginning of GCF.